legion_test/legion_test/robot/utils.py

```diff
@@ -167,16 +172,20 @@ def check_valid_http_response(url):
             raise Exception('Unexpected case happen!')
 
     @staticmethod
-    def get_component_auth_page(url):
+    def get_component_auth_page(url, token=None):
         """
         Get component main auth page
 
-        :param url: component url
-        :type url: str
+        :param str url: component url
+        :param str token: token for the authorization

```

Comment:
Function should be renamed to show that it can be used only for model invocation